actions/dependabot/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+version: 2
+updates:
```

Comment:
why do you use this if you don't have an libs to run it on?

Reply:
What do you mean?

As an example, the yaml script allows you make changes according to the need:

 version: 2 
 updates:
   - package-ecosystem: "docker"
     directory: ./anything/src/main/docker/
     schedule:
       interval: "daily"
     packages:
       - name: "nginx"
         version: "latest"
       - name: "mysql"
         version: "5.7"

Reply:
@Siegfriedk  Could you please elaborate if your question is not answered?

Reply:
ah i think i didn't get it that it should be an example config. The path looked like a normal github action

Reply:
Yes, an example config along with other tools. I have made small changes (added npm) and hence the update.

Reply:
This makes no sense. 
Actions are stored in the actions directory which can be used by workflows in the workflows directory.
The actions look different, e.g. inputs must be defined there.

Reply:
Well, I suppose I have done exactly that. All the tools with respective yaml script are now moved to actions folder. 
And also yes to your second point, which is why I simply gave an example.

Reply:
Nope, the actions directory ist a dedicated directory for reusable worklows.
You copy/pasted my non-reusable workflows.

Reply:
Maybe this helps for understanding: https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow

So technically you can put **reusable**  workflows anywhere, but it is common to put them in the actions directory.

Reply:
> Nope, the actions directory is a dedicated directory for reusable worklows. You copy/pasted my non-reusable workflows.

@scherersebastian : To educate you on raising remark on me "I copy/pasted your workflow"
Firtly, when you say, "my non-reusable workflows", if you are so specific to your OWN work, please provide your copyrights to it.
I beleive that is not the case here and seems childish to say its mine.

Secondly, Copy/paste would idealy mean, duplication of something and re-using it. In this case here, I have moved the existing script to one specific folder.
Do I have the rights to do it? Yes, of-course. Its open-source :)
This is done not just for dependabot, but also for trivy, kicks as well.